src/behaviors/loading/demo/loading.module.js

```diff
@@ -0,0 +1,16 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('semantic.ui.angular.loading',[
+            'semantic.ui.behaviors.loading'
+        ]).run(function($rootScope) {
+
+          $rootScope.opts = {
```

Comment:
this worries me a bit. We should probably not be polluting the root scope this way. How about registering a constant or variable in the module instead?